store/postgres_migration.go

```diff
@@ -204,11 +187,16 @@ func createFieldStatement(tenant string, info tableInfo, fieldInterface interfac
 func createJoinStatement(tenant string, info tableInfo, joinInterface interface{}) (string, error) {
 	join, ok := (joinInterface).(core.TableJoin)
 	if !ok {
-		return "", fmt.Errorf("uniqueInterface not assignable to bool: %s", joinInterface)
+		return "", fmt.Errorf("cannot assign type %s to core.TableJoin", reflect.TypeOf(joinInterface).String())
 	}
-	if join.Single {
-		return "ALTER TABLE IF EXISTS " + psqlAbsTableName(tenant, info.TableName) + " ADD CONSTRAINT " + info.TableName + "_" + info.ElementName + "_unique UNIQUE (" + info.ElementName + tableJoinSuffix + ");", nil
-	} else {
-		return "ALTER TABLE IF EXISTS " + psqlAbsTableName(tenant, info.TableName) + " ADD COLUMN IF NOT EXISTS " + info.TableName + tableJoinSuffix + " SERIAL;", nil
+	return "ALTER TABLE IF EXISTS " + psqlAbsTableName(tenant, info.TableName) + " ADD COLUMN IF NOT EXISTS " + join.Table + tableJoinSuffix + " SERIAL;", nil
+}
+
+// removeJoinStatement removes a the column which acts as a join for a table
+func removeJoinStatement(tenant string, info tableInfo, joinInterface interface{}) (string, error) {
+	join, ok := (joinInterface).(core.TableJoin)
+	if !ok {
+		return "", fmt.Errorf("cannot assign type %s to core.TableJoin", reflect.TypeOf(joinInterface).String())
```

Comment:
❓ The canonical logging format would be `"cannot assign type to core.TableJoin: %s"`, no?